application.go

```diff
@@ -225,11 +225,7 @@ func init() {
 			if useAwsInTests {
 				awsBucketEnvVar += "_TEST"
```

Comment:
The warning here is happening because this code is only triggered if `isGoTest` is true, but the `awsBucketEnvVar` variable is only used if `isGoTest` is false.

Reply:
Yeah, we can drop this env var.